jupyter_book/book_template/content/beta/titles.ipynb

```diff
@@ -0,0 +1,67 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Page titles and authors\n",
+    "\n",
+    "## Adding titles to your book's pages\n",
+    "\n",
+    "Jupyter Notebooks don't have an internal concept of a \"title\", but it's common\n",
+    "to want an official title for each page of your book. Jupyter Book will add titles\n",
+    "to your pages according to a few rules.\n",
+    "\n",
+    "Each of the options below is controlled by the `titles: ` parameter in your\n",
+    "book's `_config.yml` file. Here is are the options for this configuration:\n",
+    "\n",
+    "* `page_titles: None`: Do not add any titles to your pages (the default)\n",
+    "* `page_titles: toc`: Infer the titles from your Table of Contents file (`_data/toc.yml`)\n",
+    "* `page_titles: infer`: Infer the titles from each page of your book. In this case,\n",
+    "  Jupyter Book will first look in `notebook['metadata']['title']` for a title. If it\n",
+    "  does not find one, it will check if the first line of the notebook is an H1 header\n",
+    "  (starting with `# `). If either of these are found, they will be used as the title.\n",
+    "  \n",
+    "\n",
+    "## Adding authorship to your book's pages\n",
```

Comment:
I think it'd be really valuable to have a page with a listed author, just to demo the feature ! Maybe this one ?

Reply:
good idea!